backend/src/packages/users/libs/types/user-with-user-details-join.type.ts

```diff
@@ -0,0 +1,13 @@
+type UserWithUserDetailsJoin = {
+  id: number;
+  createdAt: Date;
```

Comment:
doesn't query return string?

Reply:
I need to check and in will correct in according to the query type

Reply:
Yep, abstract model has string type for createdAt and updatedAt fields so I will change this type